components/Schedule/SportClassCard.tsx

```diff
@@ -1,9 +1,56 @@
+import { useMutation } from '@apollo/client'
 import styled from 'styled-components'
+import gql from 'graphql-tag'
 import { decimalToTime } from '../../lib/dateHelpers'
+import { CURRENT_USER_QUERY } from '../UserAuth/User'
+import { useUser } from '../UserAuth/User'
+import { useModal } from '../Modals/ModalContext'
+
+const ADD_TO_BOOKINGS_MUTATION = gql`
+  mutation ADD_TO_BOOKINGS_MUTATION($id: ID!) {
+    addToBookings(sportClassId: $id) {
+      id
+    }
+  }
+`
+
+interface SportClassCardProp {
+  classData: {
+    id?: string
```

Comment:
Shouldn't `id` always be mandatory?

Reply:
I also just remembered; don't you already have these props in another `interface`? I think where I added the comment where you can `extend` interfaces.

If so: use it!